lib/ansible/module_utils/urls.py

```diff
@@ -81,6 +81,16 @@
 # agrees to be bound by the terms and conditions of this License
 # Agreement.
 
+'''
+The **urls** utils module offers a replacement for the `requests`_ library for most cases. 
+
+Sometimes you really need the requests library, but you should always check, whether 
+this utils module is sufficient for your use case.
+
+.. _requests:
+   http://docs.python-requests.org/en/master/
+'''
```

Comment:
Hmm.... I see in the original bug that ansible-modules-validate is throwing an error for using requests and that was part of what prompted this doc update.  That does make it a good idea to mention requests here.... What about something like:

```
The **urls** utils module offers a replacement for the urllib2 python library.

urllib2 is the python stdlib way to retrieve files from the Internet but it lacks some security features (around verifying SSL certificates) that users should care about in most situations.  using the functions in this module corrects deficiencies in the urllib2 module wherever possible.

There are also third-party libraries (for instance, requests) which can be used to replace urllib2 with a more secure library.  However, all third party libraries require that the library be installed on the managed machine.  That is an extra step for users making use of a module.  If possible, avoid third party libraries by using this code instead.
```